test/src/main/scala/scala/test/junit/RulesScalaIntegrationTests.scala

```diff
@@ -0,0 +1,87 @@
+package scala.test.junit
+
+import build.bazel.tests.integration.BazelBaseTestCase
+import org.junit.{Assert, Test}
+import scala.collection.JavaConverters._
+
+class RulesScalaIntegrationTests extends BazelBaseTestCase {
+
+  private val NoTestsFoundExitCode = 3
+  private val NoTestsFoundLabel = "no_tests_found"
+
+  @Test
+  def singleTestSoTargetWillNotFailDueToNoTestsTest: Unit = {
+    givenRulesScalaWorkspaceFile()
+    givenBuildFile()
+    givenEmptyTestSuitFile()
+
+    val cmd = driver.bazelCommand("test", "//...").build()
+    val exitCode = cmd.run()
+    val errorOutput = cmd.getErrorLines.asScala
+
+    assertNoTestsFoundCorrectOutput(errorOutput, exitCode)
+  }
+
+  private def assertNoTestsFoundCorrectOutput(output: Seq[String], exitCode: Int): Unit = {
+    Assert.assertEquals(exitCode, NoTestsFoundExitCode )
+    val hasCorrectOutput = output.exists(line => {
+      line.contains("FAIL") && line.contains(NoTestsFoundLabel)
+    })
+
+    Assert.assertTrue(hasCorrectOutput)
+  }
+
+  private def givenRulesScalaWorkspaceFile(): Unit = {
+    val workspace =
+      """
+        |rules_scala_version="a8ef632b1b020cdf2c215ecd9fcfa84bc435efcb" # update this as needed
```

Comment:
can we not somehow point to the current version? It feels weird to always be testing a previous version. Also, how updating this seems like a challenge especially if we have many.